nvdaHelper/interfaces/nvdaController/nvdaController.idl

```diff
@@ -26,12 +26,39 @@ cpp_quote("This license can be found at:")
 cpp_quote("http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html")
 cpp_quote("*/")
 
+typedef enum {
+/**
+ * Indicates that a speech sequence should have normal priority.
+ */
+	SPEECH_PRIORITY_NORMAL = 0,
+/**
+ * Indicates that a speech sequence should be spoken after the next utterance of lower priority is complete.
+ */
+	SPEECH_PRIORITY_NEXT = 1,
+/**
+ * Indicates that a speech sequence is very important and should be spoken right now,
+ * interrupting low priority speech.
+ * After it is spoken, interrupted speech will resume.
+ * Note that this does not interrupt previously queued speech at the same priority.
+ */
+	SPEECH_PRIORITY_NOW = 2
+} SPEECH_PRIORITY;
+
+typedef enum {
+	SYMBOL_LEVEL_NONE = 0,
+	SYMBOL_LEVEL_SOME = 100,
+	SYMBOL_LEVEL_MOST = 200,
+	SYMBOL_LEVEL_ALL = 300,
+	SYMBOL_LEVEL_CHAR = 1000,
+	SYMBOL_LEVEL_UNCHANGED = -1
+} SYMBOL_LEVEL;

```

Comment:
could you add a comment on this Enum that it must match the enum in python code.
Same with in the python definition, so that these 2 enums are kept in sync.

Reply:
It seems like these comments are missing from the python module

Reply:
Ah I'm sorry, I missed your second question there.